packages/alert/src/index.vue

```diff
@@ -8,7 +8,7 @@
     >
       <i v-if="showIcon" class="el-alert__icon" :class="[ iconClass, isBigIcon ]"></i>
       <div class="el-alert__content">
-        <span v-if="title || $slots.title" class="el-alert__title" :class="[ isBoldTitle ]">
+        <span v-if="!!title || $slots.title" class="el-alert__title" :class="[ isBoldTitle ]">
```

Comment:
Based on the logic you provided, `title` is type of string, `$slots.title` is type of function, if you wanted to test whether a string variable is truthy, you can do it in different ways:
```ts
if (string) {
  // code
}
```
or 
```ts
if (!!string) {
  // code
}
```
or 
```ts
if (isDefined(string)) {
  // code
}
```

In your case here you only performed casting with the string typed title instead of both `title` and `$slots.title`, for consistency I don't think this change is legit.